src/components/RegistrationForm.js

```diff
@@ -0,0 +1,105 @@
+import React, { PropTypes } from 'react';
+import ReactDOM from 'react-dom';
+import styled from 'styled-components';
+import SocialRegister from './SocialRegister';
+import '../App.css';
+
+export default class RegistrationForm extends React.Component {
+    state = {
+        agreeNotChecked: true,
+        emailIsEmpty: true,
+        nickIsEmpty: true,
+        passwordIsEmpty: true
+    };
+    static propTypes = {
+        email: PropTypes.string.isRequired,
+        nick: PropTypes.string.isRequired,
+        password: PropTypes.string.isRequired
+    };
+
+    onBtnClickHandler = (e) => {
+        e.preventDefault();
+        let email = ReactDOM.findDOMNode(this.refs.email).value;
```

Comment:
you really must invoke ReactDOM.findDOMNode everytime? just this.refs.email.value not working?